src/topojson/topojson.ml

```diff
@@ -315,19 +318,45 @@ module Make (J : Intf.Json) = struct
                     (decode_or_err (J.to_array (decode_or_err J.to_float)))))
               arcs
           in
-          Ok { objects; arcs }
+          let fm = foreign_members json in
+          Ok { objects = geometries; arcs; foreign_members = fm }
       | _, _ -> Error (`Msg "No objects and/or arcs field in Topology object!")
+
+    let to_json ?bbox { objects; arcs; foreign_members } =
+      J.obj
+        ([
+           ("type", J.string "Topology");
+           ("objects", J.obj objects);
+           ("arcs", J.array (J.array (J.array J.float)) arcs);
+         ]
+        @ bbox_to_json_or_empty bbox
+        @ foreign_members)
```

Comment:
While working with this function , got some issues with  `("objects", J.obj objects);` as we already know the fact that:

> A topology must have a member with the name “objects” whose value is another object. The value of each member of this object is a geometry object.

so I think `J.obj` would be correct (or J.list have to be used, got confused), hence I tried with it.
`J.obj` has type `(string * json)` list but `objects` has type `(string * Geometry.t) list` so, we indeed wanted `(string * json)` list type in this and not particularly  `(string * Geometry.t) list`....

Got confused, how to do it?

Reply:
It's essentially the inverse to what we did for `of_json` where we converted from `json` to `Geometry.t` and now we want to convert back: 

```ocaml
("objects", J.obj (List.map (fun (k, v) -> (k, Geometry.to_json v)) objects));
```

Reply:
Oh never thought the sol would be too easy, only the reverse technique has to be applied, learning so much.
Thanks Patrick:))